transaction-coordinator/src/main/java/org/apache/kafka/coordinator/transaction/TransactionLogConfig.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.coordinator.transaction;
+
+public class TransactionLogConfig {
+    // Log-level config default values
+    public static final int DEFAULT_NUM_PARTITIONS = 50;
+    public static final int DEFAULT_SEGMENT_BYTES = 100 * 1024 * 1024;
+    public static final short DEFAULT_REPLICATION_FACTOR = 3;
+    public static final int DEFAULT_MIN_IN_SYNC_REPLICAS = 2;
+    public static final int DEFAULT_LOAD_BUFFER_SIZE = 5 * 1024 * 1024;
+}
```

Comment:
Nit: We usually have a trailing line at the end of files.

Reply:
Done, and did the same to other files I created as well